### PR TITLE
Refactor language syncronizer. Remove it from the React tree, integra…

### PR DESCRIPTION
**Pull request by sneridagh**
…te it into the Api Redux middleware.

This one is much better than https://github.com/plone/volto/pull/3103

### CODE REVIEW

File: cypress/tests/multilingual/basic-multilingual.js [hunk 1]
```diff
@@ -26,11 +26,14 @@ describe('Basic multilingual Tests', () => {
     cy.url().should('eq', Cypress.config().baseUrl + '/it');
   });
 
-  it('Language coming from SSR', function () {
+  it.only('Language coming from SSR', function () {
```
Comment (tiberiuichim) [inline cypress/tests/multilingual/basic-multilingual.js:29]: it.only

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

File: cypress/tests/multilingual/basic-multilingual.js (hunk 1 above)
Comment (sneridagh) [reply cypress/tests/multilingual/basic-multilingual.js]: Done

**Comment by sneridagh** [comment]
Even with server-side support now.

**Review by giuliaghisini — COMMENTED** [review]
I just have a doubt .. what happens in the babel view?

**Comment by sneridagh** [comment]
Good question, the TranslationObject has the means for trigger the translation. The tests pass, so it works as expected.

**Review by reebalazs — APPROVED** [review]

**Review by tiberiuichim — APPROVED** [review]

**Pull request merged**